server/zanata-frontend/src/webpack.config.js

```diff
@@ -258,15 +261,6 @@ module.exports = function (env, isEditor, devServerPort) {
         // storybook should use the fallback: style-loader
         disable: storybook || dev
       }),
-      new webpack.NoEmitOnErrorsPlugin(),
-
-      prod
-        ? new webpack.optimize.UglifyJsPlugin({ sourceMap: true })
```

Comment:
I don't see anything about minimisation now. Is this just implicit for 'production' now?

Reply:
Yes, minimization and a host of other options appear to be enabled by default now , which explains how they can claim such big performance gains with webpack 4: https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a